.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
   - git -C "$TRAVIS_BUILD_DIR" checkout -b travis-testing
   - cd $DRUPAL_DIR; composer config repositories.local path "$TRAVIS_BUILD_DIR"
   - composer require "islandora/islandora_image:dev-travis-testing as dev-8.x-1.x" --prefer-source
-  - cd web; drush en -y islandora_image
+  - cd web; drush en -y media_entity media_entity_image; drush en -y islandora_image
```

Comment:
If media_entity_image is a requirement of this module, shouldn't it be stated in the .info.yml file? I feel like adding it here it the wrong place to add this dependancy.